src/powfacpy/exceptions.py

```diff
@@ -65,12 +65,22 @@ def __init__(self,obj_str):
     self.message = (f"Please activate {obj_str}.")
     super().__init__(self.message)
 
-class PFNoPlotActivatedError(PFInterfaceError):
-  """Attempt to plot, but no plot is active.
+class PFAttributeNotSpecifiedError(PFInterfaceError):
+  """Attempt to access an attribute of a powfacpy class,
+  but it was not specified.
+  """ 
+  def __init__(self,attribute_description):
+    self.message = (f"Attempt to access {attribute_description}, " 
+    "but it was not specified.")
```

Comment:
what do you mean by specified? Not given to a function via a input argument? Or doesnt exist, so not instantiated or something?

Reply:
I changed this to AttributeNotSetError. I think this is more descriptive because there are often get/set methods to get/set an attribute in object oriented programming. The error indicates that an attribute of a class instance is accessed but it was not set before (usually it is None then).